erpnext/healthcare/doctype/lab_test/lab_test.py

```diff
@@ -111,7 +111,7 @@ def create_lab_test_from_invoice(invoice_name):
 			if lab_test_created != 1:
 				template = get_lab_test_template(item.item_code)
 				if template:
-					lab_test = create_lab_test_doc(True, invoice.ref_practitioner, patient, template)
+					lab_test = create_lab_test_doc(True, invoice.ref_practitioner, patient, template, company, invoice.company)
```

Comment:
```suggestion
					lab_test = create_lab_test_doc(True, invoice.ref_practitioner, patient, template, invoice.company)
```
Extra company parameter passed. Only the invoice.company should be passed.